src/NuGet.Core/NuGet.Protocol/Resources/PackageMetadataResourceV3.cs

```diff
@@ -30,6 +37,16 @@ public class PackageMetadataResourceV3 : PackageMetadataResource
             _packageDetailsUriResource = packageDetailsUriResource;
         }
 
+        /// <summary>
+        /// Query nuget package list from nuget server for Consolidate UI. This implementation optimized for performance so instead of keeping gian JObject in memory we use strong types.
```

Comment:
```suggestion
        /// Query nuget package list from nuget server for Consolidate UI. This implementation optimized for performance, so instead of keeping giant JObject in memory, we use strong types.
```

Reply:
```suggestion
        /// Query nuget package list from nuget server for Consolidate UI.
```

The second sentence is commentary about the change made in this PR, rather than being documentation for people using the method. My criteria for comments is: would it be useful to someone in 1 year from now who never read the old code?

Reply:
Fixed.